From 4fd4abd4d99524a676f4b85a7bab978e0e0be27e Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Wed, 12 Jun 2024 22:13:02 -0700
Subject: [PATCH] Docs/process/email-clients: Document HacKerMaiL

HacKerMaiL (hkml) [1] is a simple tool for mailing lists-based
development workflows such as that for most Linux kernel subsystems.  It
is actively being maintained by DAMON maintainer, and recommended for
DAMON community[2].  Add a simple introduction of the tool on the
email-clients document, too.

[1] https://github.com/sjp38/hackermail
[2] https://lore.kernel.org/20240621170353.BFB83C2BBFC@smtp.kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/email-clients.rst | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/process/email-clients.rst b/Documentation/process/email-clients.rst
index 471e1f93fa09..71483667b06a 100644
--- a/Documentation/process/email-clients.rst
+++ b/Documentation/process/email-clients.rst
@@ -370,3 +370,12 @@ clients may encrypt responses to encrypted mail for all recipients, including
 the mailing lists.
 Unless a way to disable this "feature" is introduced, Proton Mail is unsuited
 to kernel development.
+
+HacKerMaiL (TUI)
+****************
+
+HacKerMaiL (hkml) is a public-inbox based simple mails management tool that
+doesn't require subscription of mailing lists.  It is developed and maintained
+by the DAMON maintainer and aims to support simple development workflows for
+DAMON and general kernel subsystems.  Refer to the README
+(https://github.com/sjp38/hackermail/blob/master/README.md) for details.
-- 
2.39.2

